web/auth.go

```
@@ -0,0 +1,49 @@
+		session.Set("authenticated", true)
+		session.Save()
+		c.JSON(http.StatusOK, gin.H{"success": true})
+	} else {
+		c.JSON(http.StatusUnauthorized, gin.H{"error": "无效的凭证"})
+	}
+}
+
+func AuthLogout(c *gin.Context) {
+	session := sessions.Default(c)
+	session.Clear()
+	session.Save()
```

golangci-lint (1.62.2)

Error return value of `session.Save` is not checked

(errcheck)